lms/djangoapps/instructor/views/api.py

```diff
@@ -946,6 +947,16 @@ def modify_access(request, course_id):
         'action': action,
         'success': 'yes',
     }
+    # manage student and admin transitions for license enforcement.
+    if run_extension_point('PCO_IS_LICENSED_CCX',course_id=course_id):
+        run_extension_point(
+            'PCO_STUDENT_ADMIN_TRANSITION',
+            user=user,
+            course_id=course_id,
+            rolename=rolename,
```

Comment:
Let's ask our Product Owner what we should do for the other roles such as:
![image](https://user-images.githubusercontent.com/40271196/191140750-5fab79cf-e131-4ec7-8295-aeaad01d90f7.png)
@anfbermudezme